src/main/java/org/mitre/caasd/commons/ids/TimeId.java

```diff
@@ -192,7 +192,7 @@ public byte[] randomBytes() {
 
     /**
      * @return The {@code bytes()} of this ID encoded as unpadded Base64 String (e.g.
-     *     "YlAmUqw/L7n0P0wcMfG+zg"). The returned String will be 22 characters.  The first 7 chars
+     *     "YpnxRaZ-usc_8KDibYzccw"). The returned String will be 22 characters.  The first 7 chars
```

Comment:
Be gone, URL-unsafe encodings

Reply:
Such a simple thing.  The `/` and `+` characters want to be your friend.  But they just arent